cli/tests/pcluster/validators/test_all_validators/test_scheduler_plugin_validators_are_called_with_correct_argument/scheduler_plugin.yaml

```diff
@@ -0,0 +1,180 @@
+              TimestampFormat: "%Y-%m-%d %H:%M:%S,%f"
+              NodeType: COMPUTE
+              LogStreamName: logfile.log
+      SystemUsers:
+        - Name: user1
+          EnableImds: true # optional, defaults to 'false'
+        - Name: user2
+    GrantSudoPrivileges: true
+    CustomSettings:  # here you can add any custom setting
+      ScaledownIdletime: 10
+      SchedulerType: "sched/backfill"
+      SuspendTimeout: 60
+  SchedulerQueues:
+    - Name: queue1
+      CapacityType: ONDEMAND
+      Networking:
+        SubnetIds:
+          - subnet-12345678
+      CustomSettings:  # here you can add any custom setting
+        Key1: String
+      ComputeResources:
+        - Name: computeresource1
+          InstanceType: c5.xlarge
+          CustomSettings:  # here you can add any custom setting
+            RealMemory: 185000
+        - Name: computeresource2
+          InstanceType: c4.xlarge
+      CustomActions:
+        OnNodeStart:
+          Script: https://test.tgz  # s3:// | https://
+          Args:
+            - arg1
+            - arg2
+        OnNodeConfigured:
+          Script: https://test.tgz  # s3:// | https://
+          Args:
+            - arg1
+            - arg2
+      Iam:
+        S3Access:
+          - BucketName: string1
+            EnableWriteAccess: False
+        AdditionalIamPolicies:
+          - Policy: arn:aws:iam::aws:policy/AdministratorAccess
+      Image:
+        CustomAmi: ami-12345678
+    - Name: queue2
+      ComputeSettings:
+        LocalStorage:
+          RootVolume:
+            Size: 35
+            Encrypted: true
+            VolumeType: gp2
+            Iops: 100
+          EphemeralVolume:
+            MountDir: /scratch
+      Networking:
+        SubnetIds:
+          - subnet-12345678
+        AssignPublicIp: true
+        SecurityGroups:
+          - sg-12345678
+        PlacementGroup:
+          Enabled: true
+          Id: String
+        Proxy:
+          HttpProxyAddress: https://proxy-address:port
+      ComputeResources:
+        - Name: computeresource1
+          InstanceType: c4.2xlarge
+        - Name: computeresource2
+          InstanceType: c5.2xlarge
+          MinCount: 1
+          MaxCount: 15
+          SpotPrice: 1.1
+          DisableSimultaneousMultithreading: true
+          Efa:
+            Enabled: true
+            GdrSupport: false
+      Iam:
+        InstanceProfile: arn:aws:iam::aws:instance-profile/CustomNodeInstanceProfile
+      Image:
+        CustomAmi: ami-23456789
+SharedStorage:
+  - MountDir: /my/mount/point1
+    Name: name1
+    StorageType: Ebs
+    EbsSettings:
+      VolumeId: vol-12345678
+  - MountDir: /my/mount/point2
+    Name: name2
+    StorageType: Efs
+    EfsSettings:
+      Encrypted: True
+      KmsKeyId: 1234abcd-12ab-34cd-56ef-1234567890ab
+  - MountDir: /my/mount/point3
+    Name: name3
+    StorageType: FsxLustre
+    FsxLustreSettings:
+      StorageCapacity: 3600
```

Comment:
Watch out, missing end-of-line character here.

Reply:
Thanks! added. The new line is not required in the config yaml